src/app/wallet/help/help.component.html

```diff
@@ -0,0 +1,191 @@
+          </a>
+
+          <a class="item" href="https://www.reddit.com/r/Particl/" target="_blank">
+            <div class="icon">
+              <mat-icon fontSet="partIcon" fontIcon="part-reddit"></mat-icon>
+            </div>
+            <div class="text">
+              <div class="title">Reddit</div>
+            </div>
+          </a>
+
+          <a class="item" href="https://plus.google.com/+Particl" target="_blank">
+            <div class="icon">
+              <mat-icon fontSet="partIcon" fontIcon="part-google"></mat-icon>
+            </div>
+            <div class="text">
+              <div class="title">Google+</div>
+            </div>
+          </a>
+
+          <a class="item" href="https://mastodon.social/@particl" target="_blank">
+            <div class="icon">
+              <mat-icon fontSet="partIcon" fontIcon="part-mastodon"></mat-icon>
+            </div>
+            <div class="text">
+              <div class="title">Mastodon</div>
+            </div>
+          </a>
+
+          <a class="item" href="https://www.youtube.com/c/Particl" target="_blank">
+            <div class="icon">
+              <mat-icon fontSet="partIcon" fontIcon="part-youtube"></mat-icon>
+            </div>
+            <div class="text">
+              <div class="title">Youtube</div>
+            </div>
+          </a>
+          
+          <a class="item" href="https://bitcointalk.org/index.php?topic=1835782.0" target="_blank">
+            <div class="icon">
+              <mat-icon fontSet="partIcon" fontIcon="part-bitcointalk"></mat-icon>
+            </div>
+            <div class="text">
+              <div class="title">BitcoinTalk</div>
+            </div>
+          </a>
+
+        </div>
+      </mat-card>
+
+      <div class="subtitle">Get involved <span class="tag">Contribute</span></div>
+      <mat-card class="card">
+        <div class="list section">
+
+          <a class="item" href="https://github.com/particl" target="_blank">
+            <div class="icon">
+                <mat-icon fontSet="partIcon" fontIcon="part-github"></mat-icon>
+            </div>
+            <div class="text">
+              <div class="title">GitHub</div>
+              <p class="desc">For developers</p>
+            </div>
+          </a>
+
+          <a class="item" href="https://www.transifex.com/particl/particl-website/" target="_blank">
+            <div class="icon">
+                <mat-icon fontSet="partIcon" fontIcon="part-document"></mat-icon>
+            </div>
+            <div class="text">
+              <div class="title">Transifex</div>
+              <p class="desc">For translators</p>
+            </div>
+          </a>
+
+          <a class="item" href="https://particl.wiki" target="_blank">
+            <div class="icon">
+              <mat-icon fontSet="partIcon" fontIcon="part-wikipedia"></mat-icon>
+            </div>
+            <div class="text">
+              <div class="title">Wiki</div>
```

Comment:
Is there a reason for the second Wiki entry? It has already been included int he 'Tutorials' section.

Reply:
@AllienWorks could you please address this comment.

Reply:
I understand what you mean. The first link is more like "if you need help, here's Wiki" and the second one is aimed more at contributors, to let them know that Wiki is available as a means to help out. I don't see a problem with it :)

Reply:
@AllienWorks, should I need to remove the duplicate link from the page. Or that option (link) refers to another link in future?